modules/110-istio/monitoring/prometheus-rules/versions.tpl

```diff
@@ -16,3 +16,21 @@
       labels:
         severity_level: "{{"{{$labels.alert_severity}}"}}"
         tier: cluster
+- name: d8.istio.k8sVersionsCompatibility
+  rules:
+    - alert: D8IstioVersionIsIncompatibleWithK8sVersion
+      annotations:
+        description: |
+          The current istio version `{{"{{$labels.istio_version}}"}}` may not work properly with the current k8s version `{{"{{$labels.k8s_version}}"}}`, because it officially  unsupported.
```

Comment:
```suggestion
          The current istio version `{{"{{$labels.istio_version}}"}}` may not work properly with the current k8s version `{{"{{$labels.k8s_version}}"}}`, because it is unsupported officially.
```